envoy/stats/refcount_ptr.h

```diff
@@ -115,7 +116,7 @@ template <class T> class RefcountPtr {
 // forth.
 #if __cplusplus < 202002L
 template <class T> static bool operator==(std::nullptr_t, const RefcountPtr<T>& a) {
-  return a == nullptr;
+  return a == nullptr; // NOLINT(clang-analyzer-cplusplus.Move)
```

Comment:
weird, why this line need this nolint...

Reply:
no idea tbh - but confirmed its necessary

```console
./envoy/stats/refcount_ptr.h:119:10: error: Method called on moved-from object 'rp4' [clang-analyzer-cplusplus.Move,-warnings-as-errors]
  return a == nullptr;
         ^
```

https://dev.azure.com/cncf/envoy/_build/results?buildId=150324&view=logs&j=b7634614-24f3-5416-e791-4f3affaaed6c&t=d565ee52-b888-5d80-7fea-dfaed104c390&l=35

Reply:
also don't know why, orz.